ci-operator/step-registry/openshift-qe/workers-infra-workload/openshift-qe-workers-infra-workload-commands.sh

```diff
@@ -543,6 +543,17 @@ EOF
 #           OPENSHIFT_WORKLOAD_NODE_INSTANCE_MEMORYSIZE=64Gi
 #-------------------------------------------------------------------------------------
 
+# For disconnected or otherwise unreachable environments, we want to
+# have steps use an HTTP(S) proxy to reach the API server. This proxy
+# configuration file should export HTTP_PROXY, HTTPS_PROXY, and NO_PROXY
+# environment variables, as well as their lowercase equivalents (note
+# that libcurl doesn't recognize the uppercase variables).
+if test -f "${SHARED_DIR}/proxy-conf.sh"
+then
+	# shellcheck disable=SC1090
+	source "${SHARED_DIR}/proxy-conf.sh"
```

Comment:
we might need to add this to kube-burner scripts as well, lets monitor test run.

Reply:
That is a plan. If that works - I will add this into all kube-burner scripts.

Reply:
It looks like if it is in `pre` step in workflow - then it will be working for test phase.
Please check [this rehearse](https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/openshift_release/47673/rehearse-47673-pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-gcp-4.15-nightly-x86-control-plane-private-3nodes/1747715346200006656) lines 66-76